src/Akka.Persistence.Sql/Journal/Types/JournalRow.cs

```diff
@@ -30,7 +30,7 @@ public sealed class JournalRow
 
         public string[] TagArr { get; set; }
 
-        public Guid? WriteUuid { get; set; }
+        public string WriterUuid { get; set; }
```

Comment:
LGTM - probably easier to work with strings than Guids. We don't gain much from a Db perspective using a strict type here.